league/views.py

```diff
@@ -1482,9 +1482,35 @@ def division_update_wont_play(request, division_id):
         'event_pk': event.pk,
         'division': division,
     }
-    print(event.pk)
     return render(request, 'league/admin/update_wont_play.html', context)
 
+@login_required()
+def division_create_forfait(request, division_id):
```

Comment:
before we merge this, can we find and replace forfait with forfeit? :D

Reply:
Indeed, good catch. Damn, those frenchies...

Reply:
whoops ^^